src/style.css

```diff
@@ -2,2 +1,76 @@
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
```

Comment:
The font-size is set to 12px for the 'html' element. This might not be responsive across different devices. Consider using relative units like 'em' or 'rem' for better scalability.